games-fps/chocolate-doom/chocolate_doom-3.0.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="https://www.chocolate-doom.org/"
+COPYRIGHT="Id Software Inc. 1993-1996
+	Simon Howard
+	James Haley
+	Samuel Villarreal"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.chocolate-doom.org/downloads/$portVersion/chocolate-doom-$portVersion.tar.gz"
+CHECKSUM_SHA256="73aea623930c7d18a7a778eea391e1ddfbe90ad1ac40a91b380afca4b0e1dab8"
+SOURCE_DIR="chocolate-doom-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chocolate_doom$secondaryArchSuffix = $portVersion
+	cmd:chocolate_doom$secondaryArchSuffix = $portVersion
+	cmd:chocolate_doom_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_heretic_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_heretic$secondaryArchSuffix = $portVersion
+	cmd:chocolate_hexen_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_hexen$secondaryArchSuffix = $portVersion
+	cmd:chocolate_server$secondaryArchSuffix = $portVersion
+	cmd:chocolate_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_strife_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_strife$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libSDL2_net_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2$secondaryArchSuffix
+	devel:libsdl2_mixer$secondaryArchSuffix
+	devel:libsdl2_net$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:install
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd src
+	mkdir -p $binDir
+	cp chocolate-doom \
+		chocolate-doom-setup \
+		chocolate-hexen \
+		chocolate-hexen-setup \
+		chocolate-heretic \
+		chocolate-heretic-setup \
+		chocolate-server \
+		chocolate-setup \
+		chocolate-strife-setup \
+		chocolate-strife $binDir
```

Comment:
When you add the [commandBinDir](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable) bloc, you might wish to change `INSTALL()` to:
```diff
 INSTALL()
 {
 	cd src
-	mkdir -p $binDir
+	install -d $commandBinDir
+	install -t $commandBinDir \
-	cp chocolate-doom \
+		chocolate-doom \
 		chocolate-doom-setup \
 		chocolate-hexen \
 		chocolate-hexen-setup \
 		chocolate-heretic \
 		chocolate-heretic-setup \
 		chocolate-server \
 		chocolate-setup \
 		chocolate-strife-setup \
-		chocolate-strife $binDir
+		chocolate-strife
 }
```